Utilities/WDLs/DownsampleAndCollectCoverage.wdl

```diff
@@ -104,18 +104,25 @@ task CollectWgsMetrics {
         ~{"-INTERVALS " + coverage_intervals} \
         -OUTPUT ~{output_basename}.wgs_metrics \
         -READ_LENGTH ~{read_length} \
-        -USE_FAST_ALGORITHM ~{use_fast_algorithm}
+        -USE_FAST_ALGORITHM ~{use_fast_algorithm} \
+        -COUNT_UNPAIRED true
+
+        cat ~{output_basename}.wgs_metrics | grep -v -e '^#' -e "^$" "~{output_basename}.wgs_metrics" | head -2 > wgs.tsv
```

Comment:
This line shouldn't have the `cat` statement since `grep` is reading directly from the file